BREAKING.md

```diff
@@ -1,10 +1,25 @@
 ## 0.43 Breaking changes
 
 - [TinyliciousClient no longer static](#TinyliciousClient-no-longer-static)
+- [Property removed from the Container class](#Property-removed-from-the-Container-class)
+- [The legacy way of creating containers has been deprecated](#The-legacy-way-of-creating-containers-has-been-deprecated)
+- [The legacy way of creating documents has been deprecated](#The-legacy-way-of-creating-documents-has-been-deprecated)
 
 ### TinyliciousClient no longer static
 `TinyliciousClient` global static property is removed. Instead, object instantiation is now required.
 
+### Property removed from the Container class
+- the `existing` property from `Container` has been removed.
+
+### The legacy way of creating containers has been deprecated
+- When using `Container.load` to create a container as a fallback when the snapshot does not exist has been deprecated and is no longer supported.
```

Comment:
I'm a bit confused by reference to snapshot here. I see two mostly orthogonal things:
1. creating containers through Container.load() flow. This is being deprecating, and we are leaving support for time being using this more explicit (but still deprecated) flow.
2. Opening files without snapshot. We only need to support it in context of above, as existing production flows have been creating containers using detached flow for very long time, and files without snapshot means really old file that did not summarize for some reason and thus 99.99% is broken (with exception of # 1, of course).

I'm not sure if even mentioning snapshot here is useful, as the key of the change - container.load() changes its behavior for non-existing files.

Reply:
I will reword this.